docs/how-to-use-playground.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta, Description } from '@storybook/addon-docs';
+import playgroundLandingPage from './images/playground-landing-page.png'
+
+<Meta
+  title="Documentation/How To Use Playground"
+  parameters={{
+    viewMode: 'docs',
+    previewTabs: {
+      canvas: {
+        hidden: true,
+      },
+    },
+  }}
+/>
+
+# How to use playground?
+Playground is a demonstration area for developers. It stays under the `playground` folder in the root. It helps you to test your features & components by eliminating the need for stories.
+
+<bl-alert variant="warning" caption="Note">You should not commit the changes in the playground folder.</bl-alert>
```

Comment:
```suggestion
<bl-alert variant="warning" caption="Note">playground folder is ignored from Git index except template.html file. It's not intended to make changes on template.html file unless you want to suggest something on template that you consider it will be helpful for everyone.</bl-alert>
```

And maybe we can move this alert below the next paragraph.